src/docs/patterns/lists.mdx

```diff
@@ -1,56 +1,193 @@
+import GlobalActionPanelHeaderImage from '../../app/assets/design-patterns/lists/action-button-in-panel-header.png';
+import GlobalActionAsButtonImage from '../../app/assets/design-patterns/lists/action-button-as-button.png';
+import SortableListImage from '../../app/assets/design-patterns/lists/sortable-list.gif';
+import LocalListActionsImage from '../../app/assets/design-patterns/lists/local-list-actions.png';
+import MultiSelectImage from '../../app/assets/design-patterns/lists/multi-select-list.png';
+import NavigateListImage from '../../app/assets/design-patterns/lists/navigate-list.png';
+
+<!-- Specs -->
+
+import Spec1Image from '../../app/assets/design-patterns/lists/spec-1.png';
+import Spec2Image from '../../app/assets/design-patterns/lists/spec-2.png';
+import Spec3Image from '../../app/assets/design-patterns/lists/spec-3.png';
+import Spec4Image from '../../app/assets/design-patterns/lists/spec-4.png';
+import Spec5Image from '../../app/assets/design-patterns/lists/spec-5.png';
+import SpecTypographyImage from '../../app/assets/design-patterns/lists/spec-typography.png';
+
+<!-- KEYWORDS: design pattern list status sort table drag drop action data -->
+
+# Lists & Tables
+
+<ImageGrid images={[ListBannerImage]} regularWidth />
+
+Lists and tables are frequently used throughout PX Blue applications. Typically, lists are used to represent groups of similar data. They can display from simple name-value pairs in a sidebar all the way to complex timelines or device lists.
 
 <MaterialDesignDescription title={'Lists'} url={'https://material.io/components/lists/'} />
 
+<!--
+A TOC like this would be nice, but apparently these links are not working
+-   [Common Use Cases](#common-use-cases)
+-   [Variations](#variations)
+-   [Behaviors](#behaviors)
+-   [Design Specifications](#design-specifications)
+-   [Developers](#developers)
+-->
+
+<Divider />
+
+# Common Use Cases
+
+<ImageGrid
+    images={[
+        NotificationListDesktopImage,
+        NotificationListMobileImage,
+        DashboardListDesktopImage,
+        DashboardListMobileImage,
+    ]}
+    caption={[
+        'Notifications and status list are the two most common use case for the list pattern.',
+        '1&2: Notifications, for mobile and desktop respectively.',
+        '3&4: A status list embedded inside a dashboard, for mobile and desktop respectively.',
+    ]}
+/>
+
+The scenarios in which lists may occur include:
+
+-   Dashboard / current status
+-   Notifications
+-   Inventories, list of assets
+-   Contacts
+-   File tree
+-   Logs
+-   Settings
+
+Lists should **NOT** be used where:
+
+-   There are no comparable data presents, such as data sets with different structures;
+-   The users demand a summary rather than detailed data value, such as on the home screen. Use [Data visualization](/patterns/visualizations) to display trends, or [heroes](https://pxblue-components.github.io/react/?path=/info/components-hero--get-read-me-story) to display a summary.
```

Comment:
I am trying to find a way to say "don't show a bulky device log on the home screen when you should show summaries"

Reply:
And what is a logical item?

Reply:
Yeah, this differentiation needs a little work. A 'logical item' is like something that matches with your mental model of a 'thing' — like a 'device' or an 'alarm' or a 'channel' rather than something more abstract. But rows in a table could also be a logical item like a 'User'.

Reply:
I don't know if the part about bulky lists needs to be said. Maybe your application is very log heavy or notification heavy and the homescreen is a big long list of something. I think that really comes down to the product and the design. I don't know if we can make a blanket statement like "Don't use a long list on a home screen"